spec/controllers/projects_controller_spec.rb

```diff
@@ -0,0 +1,92 @@
+require 'rails_helper'
+
+RSpec.describe ProjectsController, type: :controller do
+  let!(:project) { create(:project) }
+  let!(:projects) { create(:project, user_id: 5)}
+  let!(:user) { create(:user, id: 5) }
+  before :each do
+    sign_in(user)
+  end
+  describe 'GET #index' do
+    context 'anonymous user' do
+      before :each do
+        login_with nil
+        get :index
+      end
+
+      it 'redirects to signin page' do
+        expect(response).to redirect_to(new_user_session_path)
+      end
+    end
+
+    context 'logged user'  do
+      before :each do
+        get :index
+      end
+
+      it 'returns success response' do
+        expect(response).to be_success
+      end
+
+      it 'assigns project' do
+        expect(assigns(:projects)).to eq [project]
+      end
+
+      it 'renders index' do
+        expect(response).to render_template(:index)
+      end
+    end
+  end
+
+  describe "DELETE #destroy" do
+    context 'user project owner' do
+      it 'removes project' do
+        delete :destroy, id: projects.id
+        binding.pry
+        expect(response).to be_success
+      end
+    end
+    context 'user not project owner' do
+      it 'does not remove project' do
+        user = create(:user, id: 6)
+        sign_in(user)
+        delete :destroy, id: projects.id
+        expect(response).to be_false
+      end
+    end
+  end
+  
+  describe "PUT #update" do
+    context 'user project owner' do
+      it 'update project' do
+        delete :update, id: projects.id
+        binding.pry
+        expect(response).to be_success
+      end
+    end
+    context 'user not project owner' do
+      it 'does not update project' do
+        user = create(:user, id: 6)
```

Comment:
I create user with wrong id. Project have user_id 5. Here i create user with id 6 to create error